community/modules/compute/schedmd-slurm-gcp-v6-partition/variables.tf

```diff
@@ -189,10 +189,18 @@ variable "nodeset_dyn" {
 # tflint-ignore: terraform_unused_declarations
 variable "network_storage" {
   description = "DEPRECATED"
-  type        = any
-  default     = null
+  type = list(object({
+    server_ip             = string,
+    remote_mount          = string,
+    local_mount           = string,
+    fs_type               = string,
+    mount_options         = string,
+    client_install_runner = map(string)
+    mount_runner          = map(string)
+  }))
+  default = []
   validation {
-    condition     = var.network_storage == null
-    error_message = "network_storage in partition module is deprecated and should be removed."
+    condition     = length(var.network_storage) == 0
+    error_message = "network_storage in partition module is deprecated and should not be set."
```

Comment:
sorry, I don't really think this is the right thing to do.
I was very happy to see this comment here. I recently saw this error message and almost opened a bug.  
This message should say *where* it is valid to set network storage. Is this now in the node_set? is it in the controller?